src/platform/plugins/shared/discover/public/application/main/state_management/redux/types.ts

```diff
@@ -17,34 +17,10 @@ import type { UnifiedSearchDraft } from '@kbn/unified-search-plugin/public';
 import type { UnifiedHistogramVisContext } from '@kbn/unified-histogram';
 import type { ESQLEditorRestorableState } from '@kbn/esql-editor';
 import type { TabItem } from '@kbn/unified-tabs';
+import type { DiscoverSession } from '@kbn/saved-search-plugin/common';
 import type { DiscoverAppState } from '../discover_app_state_container';
 import type { DiscoverLayoutRestorableState } from '../../components/layout/discover_layout_restorable_state';
 
-export enum LoadingStatus {
-  Uninitialized = 'uninitialized',
-  Loading = 'loading',
-  LoadingMore = 'loading_more',
-  Complete = 'complete',
-  Error = 'error',
-}
-
-type RequestState<
-  TResult extends {},
-  TLoadingStatus extends LoadingStatus = Exclude<LoadingStatus, LoadingStatus.LoadingMore>
-> =
-  | {
-      loadingStatus: Exclude<TLoadingStatus, LoadingStatus.Error>;
-      result: TResult;
-    }
-  | {
-      loadingStatus: LoadingStatus.Error;
-      error: Error;
-    };
-
-export type DocumentsRequest = RequestState<DataTableRecord[], LoadingStatus>;
-export type TotalHitsRequest = RequestState<number>;
-export type ChartRequest = RequestState<{}>;
```

Comment:
We never ended up using these so I removed them. They can be re-added later when needed.